Assets/Scripts/SS3D/Systems/Entities/Debug/MindSwapDebug.cs

```diff
@@ -9,20 +10,33 @@ public class MindSwapDebug : NetworkActor
     {
         public Entity Origin;
         public Entity Target;
+        private Controls.OtherActions _controls;
+
+        private void OnEnable()
+        {
+            try
```

Comment:
replace with` if(null && !didstart) return; else { Debug.Error(); return; }` So we can still have an error in case performed is null later in the game.